packages/crayons-core/src/components/select/select.tsx

```diff
@@ -231,6 +231,17 @@ export class Select {
    * Describes the select's boundary HTMLElement
    */
   @Prop() boundary: HTMLElement;
+  /**
+   * Props to be passed for creatable select
+   * isCreatable: boolean - If true, select accepts user input as selected option
```

Comment:
can you be more explicit saying , If true, select accepts user input that are not present as options and add them as options ?